extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -341,11 +331,9 @@ export class AmpAnalytics extends AMP.BaseElement {
     if (this.iframeTransport_) {
       return;
     }
-    const TAG = this.getName_();
-    const ampAdResourceId = user().assertString(
-        getAmpAdResourceId(this.element, getTopWindow(this.win)),
-        `${TAG}: No friendly amp-ad ancestor element was found for ` +
-        'amp-analytics tag with iframe transport.');
+    const url = getIframeTransportScriptUrl(this.getAmpDoc().win);
```

Comment:
Nit, having url as its own variable is unnecessary:
this.preconnect.preload(getIframe....)